src/main/resources/templates/fragments/reusable-components.html

```diff
@@ -1,11 +1,12 @@
+<!DOCTYPE html>
 <html xmlns:th="http://www.thymeleaf.org">
 <div th:fragment="navbar">
     <nav class="navbar navbar-dark bg-primary">
         <div class="container-fluid">
             <a class="navbar-brand" href="/">Hobby Collection</a>
             <form class="d-flex">
                 <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
-                <button class="btn btn-success" type="submit">Search</button>
+                <button class="btn btn-secondary" type="submit">Search</button>
```

Comment:
This button wasn't the same color as the rest, but it was similar enough that I thought it was a little difficult to see, and it also didn't match the color scheme of the website. I switched this to match the existing buttons in the modals and the other button I fixed. We want the contrast to be sufficient enough that our users can tell the difference between the background and foreground elements.

Citation: https://www.w3.org/WAI/tips/designing/#provide-sufficient-contrast-between-foreground-and-background